env/work/srw_python/SRWLIB_Example16.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
```

Comment:
Example 16 updated - now it saves `.dat` files to `data_example_16/` and processes the case when scipy is not installed.